docsite.html

```diff
@@ -488,18 +492,19 @@ <h2 class="h2">Floating Action Button</h2>
                 <p class="py-2 small-text">
                     A floating action button (FAB) performs the primary, or most common, action on a screen. It appears
                     in front of all screen content, typically as a circular shape with an icon in its center.
-                    You can use FAB by adding <span class="primary-text-color">"btn-fab"</span> as child class name inside anchor parent tag.
+                    You can use FAB by adding <span class="primary-text-color">"btn-fab"</span> as child class name
+                    inside anchor parent tag.
```

Comment:
Code indentation could be improved here in this way, as that helps in code readability and maintainability.

```HTML
<p class="py-2 small-text">
    A floating action button (FAB) performs the primary, or most common, action on a screen.
    It appears in front of all screen content, typically as a circular shape with an icon in its center.
    You can use FAB by adding 
    <span class="primary-text-color">"btn-fab"</span>
    as child class name inside anchor parent tag.
</p>
```